packedForest/test/fpTests/rfTreeTest.h

```diff
@@ -0,0 +1,48 @@
+#include "../../src/forestTypes/basicForests/rfClassification/fpForestClassificationBase.h"
+#include "../../src/forestTypes/basicForests/rfClassification/rfTree.h"
+#include "../../src/fpSingleton/fpSingleton.h"
+#include <vector>
+
+TEST(testRFtree, testOOB){
+  float A = 0;
+  float B = 0;
+	float totalCorrect = 0;
```

Comment:
at least github thinks the indents are off in this file

Reply:
I reformatted with vscode.